modules/interactive_music/audio_stream_interactive.cpp

```diff
@@ -1018,6 +1018,10 @@ int AudioStreamPlaybackInteractive::get_loop_count() const {
 }
 
 double AudioStreamPlaybackInteractive::get_playback_position() const {
+	if (playback_current != -1) {
+		State const &current_state = states[playback_current];
```

Comment:
```suggestion
		const State &current_state = states[playback_current];
```

Godot style

Reply:
Thanks for the feedback! Fixed and squashed the change